### PR TITLE
Deal with MacOS plist issue

### PR DESCRIPTION
**Pull request by cyclemaxwell**
MacOS applications must provide a reason for requesting bluetooth access in a plist file. Use of a plist file requires creation of an app bundle. This pull request includes a `.spec` file for use with `PyInstaller`, includes an `__init__.py` file to make python recognize the openhrv folder as a module, a bugfix because MacOS does not reveal bluetooth addresses but instead uses UUIDs, and supporting documentation.

### CODE REVIEW

File: openhrv/view.py [hunk 1]
```diff
@@ -348,7 +349,10 @@ def connect_sensor(self):
         address = self.address_menu.currentText().split(",")[1].strip()
         if not valid_address(address):
             print(f"Invalid sensor address: {address}.")
-            return
+            if platform == "darwin":
```
Comment (JanCBrammer) [inline openhrv/view.py:352]: That's interesting, you uncovered a some dead code:
https://github.com/JanCBrammer/OpenHRV/blob/2b0d3ecb72d7be3cf594b1533772b3f0f6ad5715/openhrv/utils.py#L15-L19

The lines above date back to when `OpenHRV` used `bleak` for the sensor connection. On `bleak`, the sensor address format is (used to be?) platform-specific: on Windows and Linux `bleak` uses the MAC format (https://en.wikipedia.org/wiki/MAC_address), whereas on MacOS it uses the UUID format (https://en.wikipedia.org/wiki/Universally_unique_identifier).

`Qt` on the other hand treats all addresses as MACs.

I will remove this guard clause and the platform-specific logic in `utils.py` soon, it's out of scope of this PR.

**Review by JanCBrammer — CHANGES_REQUESTED** [review]
@cyclemaxwell, thanks for the super-thorough PR!

Only one nitpick before I merge this: could you remove the two `.DS_Store` files and add `.DS_Store` to the `.gitignore`?

**Comment by JanCBrammer** [comment]
@cyclemaxwell, I'm merging this, thanks again for the PR! 

Note that I've removed the MacOs-specific logic that guards the address validation on MacOs. 
I think it's best to deal with the address validation on MacOs in a separate issue as you suggest.

I'd be grateful if you could open an issue regarding the address validation on MacOs (I'm lacking a bit of context regarding all things Mac, since I don't have access to a machine).

**Review by JanCBrammer — APPROVED** [review]

**Comment by cyclemaxwell** [comment]
@JanCBrammer, thank you! I'll open an issue soon for the address vs uuid problem.

**Pull request merged**